Exercise_D_E/index.html

```diff
@@ -8,9 +8,49 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="./assets/style/index.css" />
   </head>
-  <body>
-    <!-- CODE HERE -->
+  <body class="fullPage">
+     <!-- CODE HERE -->
 
-    <script src="" async defer></script>
+    <div class="upperDiv">
+      <img class="flowershopPic" src="./assets/images/flowershop.jpg" />
+        <div class="Beautiful">
+          <header>Beautiful Flowers</header>
+          <br />
+          <h1 class="Learn">Learn more</h1>
+        </div>
+    </div>
+
+    <div class="lowerGreen" style="background-color: #b9cf88; height: 400px">
+      <br />
+      <br />
+      <br />
+      <br />
+
+      <table class="table" cellspacing="30">
+        <tr>
+          <td>
+            <img src="./assets/images/flower-icon.png" width="50px" />    Flowers
+            for every kind of situation
+          </td>
+          <td>
+            <img src="./assets/images/payment-icon.png" width="50px" />    All major
+            payments kinds accepted
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <img src="./assets/images/phone-icon.png" width="50px" />    Free
+            consultation
+          </td>
+          <td>
+            <img src="./assets/images/truck-icon.png" width="50px" />    City wide
+            delivery
+          </td>
+        </tr>
+      </table>
+
+
+    <!-- <script src="" async defer></script> -->
   </body>
 </html>
```

Comment:
For this exercise, you would have to add in the elements and you would have to add the Flowershop Logo with the company name on the top. In addition to that, you would need to increase the size of the background. You can refer to the answers to see how to change them. 

Here are some adjustments that could be made to your code: 
`
.flowershopPic {
  /* background-image: url("./assets/images/flowershop.jpg"); */
  filter: brightness(40%);
  width:100%;
}
`